### PR TITLE
[WIP]: Backend testing

### PR DESCRIPTION
**Pull request by bytewiz**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/app.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ app.use(
 app.use(cookieParser());
 app.use(cors());
 
-app.use(process.env.API_PATH, apiRouter);
+app.use('/api', apiRouter);
```
Comment (tpetrina) [inline src/server/app.js:39]: Why this change?

**Review by tpetrina — APPROVED** [review]

**Comment by benna100** [comment]
There are conflicts here

**Comment by tpetrina** [comment]
Take inspiration from my branch: https://github.com/HackYourFuture-CPH/chattie/compare/develop...server-tests. It also contains migrations and seeds for testing. We could even have special seeds for testing this way!

**Comment by dpfernandes** [comment]
closing PR after confirming with @bytewiz
